JSDemos/Demos/Toast/Overview/Vue/App.vue

```diff
@@ -15,6 +15,7 @@
       v-model:visible="isVisible"
       v-model:message="message"
       v-model:type="type"
+      :display-time="600"
```

Comment:
It would still work but if we're going to follow ESLint, it expects it to be enclosed by double quotes.
![image](https://github.com/DevExpress/devextreme-demos/assets/133762589/5981d89c-3e7c-4aa3-b877-55f71158dc74)

Reply:
summary: we've resolved this. Actually, the linter asks for a semicolon here, so let's leave this as it is